### PR TITLE
Optimized check for value presence in run container

### PR DESCRIPTION
**Pull request by xtonik**
### SUMMARY
Optimized performance of `(Mappeable)RunContainer.contains(char)` via some minor refactorization and use expression `w &= w - 1` instead of `w ^= w & -w` for iterating by set bits.

| Benchmark | Mode  | Cnt | Score   | Error    | Units  |
|-----------|-------|-----|---------|----------|--------|
| optimized | thrpt | 5   | 386.767 | ± 92.655 | ops/ms |
| original  | thrpt | 5   | 275.023 | ± 41.415 | ops/ms |

To be able to compare validity of original version and newly proposed one, I **have to add another methods `contains2()` to production code** , so if this PR will be accepted, **these methods including validity test and benchmark** (there are others for benchmarking `contains()`) **have to be removed first**. I will do that of course.

### Automated Checks

- [x] I have run `./gradlew test` and made sure that my PR does not break any unit test.
- [x] I have run `./gradlew checkstyleMain` or the equivalent and corrected the formatting warnings reported.

### CODE REVIEW

File: RoaringBitmap/src/main/java/org/roaringbitmap/buffer/MappeableRunContainer.java [hunk 1]
```diff
@@ -2727,6 +2727,63 @@
...
+    int start = getValue(0);
+    int stop = start + getLength(0);
+    while(ib < MappeableBitmapContainer.MAX_CAPACITY / 64 && ir < runCount) {
+      long w = bitmapContainer.bitmap.get(ib);
+      while (true) {
+        long r = ib * 64 + Long.numberOfTrailingZeros(w);
+        if (r < start) {
+          return false;
+        } else if (r > stop) {
+          ++ir;
```
[the remark this answers is not in the document]
Comment (xtonik) [reply RoaringBitmap/src/main/java/org/roaringbitmap/buffer/MappeableRunContainer.java:2745]: False alarm as related variable `ib` is limited form top by threshold `MappeableBitmapContainer.MAX_CAPACITY / 64`, maximum value evaluated by the multiplication is 1023*64 = 65472.

**Comment by lemire** [comment]
Looks sensible. Can you clean your code prior to merging?

**Comment by xtonik** [comment]
Unnecessary staff was cleaned up.

**Comment by lemire** [comment]
Merging.

**Pull request merged**